cypress/integration/ui/create_dashboard/index.js

```diff
@@ -28,11 +28,15 @@ const toggleShowMoreButton = () => {
 }
 
 beforeEach(() => {
-    cy.visit('/')
+    cy.visit('/', {
+        timeout: 10000,
```

Comment:
Consider moving the `timeout: 10000` to a util const for so it can be imported and easily be configured (e.g. to use a different timeout when using stubs) and to prevent repetition.

Reply:
I will. There will be some more changes coming and I'll address this in an upcoming PR.